### ISSUE TITLE
 How to set up a proxy?

### ISSUE THREAD

**Issue by chenyanmo** [taylorbobaylor/TrailerDownloader#16]
Running in docker, is it possible to set http_proxy

log:

The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing. | stdout
-- | --

**Comment by taylorbobaylor** [comment]
No, there's no way to do that currently. Are there any other errors because it shouldn't be taking longer than the default I'd think.
